packages/ldes-postgres-connector/template.ts

```diff
@@ -0,0 +1,21 @@
+export const template = {
```

Comment:
Maybe you can add a type to this object like so:

```
export const template: IConfigTemplate = {
```

Reply:
Done !